src/web/views.py

```diff
@@ -42,4 +67,7 @@ class GetTechGroups(GenericHtmxView):
 class GetEvents(GenericHtmxView):
     """Get a list of upcoming Events and render a partial to use on the sidebar navigation"""
     template_name = "web/partials/sidebar_items.htm"
-    queryset = Event.objects.filter(date_time__gte=timezone.now())
+
+    def __init__(self, **kwargs):
```

Comment:
super-ing should not be necessary here; looks like I borked the GenericHtmxView; there should be an __init__ there. I'll fix that and update this after.

Reply:
I just added the `super` as a best-practice, though it might not be necessary for this base class.

I moved the `queryset` into `__init__` because otherwise `timezone.now()` is only calculated once when the class is first parsed, since it was a class-level variable, not an instance variable.